app/styles/ember-searchable-select/style.scss

```diff
@@ -33,6 +33,16 @@ $searchable-select-spacer-horizontal: 1em;
       border-bottom-right-radius: 0;
     }
 
+    &--multiple {
+      background: none;
+      padding-top: 0.7em;
+      min-height: 41/14*1em;
```

Comment:
I noticed a bunch of css like this. x/y*1em. Interested to hear more about this.